src/FsUnit.NUnit/FsUnit.fs

```diff
@@ -45,7 +45,7 @@ module TopLevelOperators =
         else
             Assert.That(y, c)
 
-    let equal x = EqualConstraint(x)
+    let equal x = Equality.IsEqualTo(x)

```

Comment:
This is breaking changes, if we merge this we will not be able use `FsUnit.fs` using paket (without referencing NuGet package)
- http://fsprojects.github.io/FsUnit/Paket.html
- http://fsprojects.github.io/FsUnit/FsUnitTyped.html#FsUnitTyped-with-Paket

Reply:
So this type would need to be moved/copied into FsUnit.fs and FsUnitTyped.fs ?

Reply:
Just double-check `Lightweight FsUnit with Paket`  guide and it is already broken by this line -https://github.com/fsprojects/FsUnit/blob/master/src/FsUnit.NUnit/FsUnit.fs#L6